doc/usecases/kdb/UC_uptodate_config.md

```diff
@@ -0,0 +1,24 @@
+# Use Case: Keeping Configuration Up-to-date
+
+## Summary
+
+- **Scope:** `libelektra-kdb`
+- **Level:** Developer Goal
+- **Actors:** Application, Elektra
+- **Brief:** Application wants up-to-date configuration data
+
+## Scenarios
+
+- **Precondition:** -
+- **Main success scenario:** Notifications
+  - Application requests asynchronous callbacks via the notification system
+  - [Application loads configuration](UC_load_config.md)
+  - Every time the configuration changes afterwards, Elektra sends a notification to Application
```

Comment:
```suggestion
  - Every time the configuration changes, Elektra sends a notification to Application(s)
```

Reply:
```suggestion
  - Every time the configuration changes, Elektra sends a notification to Application
```

The "(s)" is wrong. I used "Application" as a proper noun here. It's the name of the actor, i.e. the placeholder for _the one application_ that's "talking" to Elektra in this use case. That's why it's capitalized and there is no "the".